pandas/_libs/tslibs/offsets.pyx

```diff
@@ -2506,6 +2506,19 @@ cdef class BQuarterEnd(QuarterOffset):
     startingMonth = 2 corresponds to dates like 2/28/2007, 5/31/2007, ...
     startingMonth = 3 corresponds to dates like 3/30/2007, 6/29/2007, ...
 
+    Parameters
+    ----------
+    n : int, default 1
+        The number of years represented.
```

Comment:
is this years or quarters?

Reply:
thanks, @MarcoGorelli , you are right.  n represents quarters, my mistake. I''l correct this.